src/pybind/mgr/dashboard_v2/tests/test_block_pool.py

```diff
@@ -2,9 +2,10 @@
 
 from __future__ import absolute_import
 
-from .helper import ControllerTestCase, authenticate
+from .helper import ControllerTestCase, authenticate, skip_if_loaded_by_mgr
 
 
+@skip_if_loaded_by_mgr
```

Comment:
same for this file - and all the other files, really, that were modified solely to add this decorator and its import: move them all to a a different patch.